src/main/java/com/farion/onlinebookstore/util/ParameterNames.java

```diff
@@ -4,4 +4,7 @@ public class ParameterNames {
     public static final String TITLE = "title";
     public static final String AUTHOR = "author";
     public static final String ISBN = "isbn";
+
+    private ParameterNames() {
+    }
```

Comment:
we can also throw an exception inside it to make sure it won't be created even inside itself